docs/general/ddl/create-table.rst

```diff
@@ -147,15 +174,78 @@ Column names are restricted in terms of patterns:
     :ref:`subscript notation <sql_dql_object_arrays>` (e.g. ``col['id']``) are
     not allowed.
 
-Advanced use
-============
 
-Tables can be:
+.. _ddl-create-table-configuration:
+
+Table configuration
+===================
+
+You can configure tables in many different ways to take advantage of the range
+of functionality that CrateDB supports. For example:
+
+.. rst-class:: open
+
+- You can :ref:`partition <partitioned-tables>` a table into one or more
+  partitions with the :ref:`PARTITIONED BY <sql-create-table-partitioned-by>`
+  clause. You control how tables are partitioned by specifying one or more
+  :ref:`partition columns <gloss-partition-column>`. Each unique combination of
+  partition column values results in a new partition.
+
+  By partitioning a table, you can segment some :ref:`SQL statements
+  <gloss-statement>` (e.g., those used for :ref:`table optimization
+  <optimize>`, :ref:`import and export <importing_data>`, and :ref:`backup and
+  restore <snapshot-restore>`) by constraining them to one or more partitions.
+
+  .. SEEALSO::
+
+      `How-to guides: Tuning partitions for insert performance`_
+
+- You can split partitions into one or more :ref:`shards <ddl-sharding>` with
```

Comment:
hmm. the doc on sharding says (first sentence): 

> Every table partition is split into a configured number of shards.

my reading of this was that every table technically has "one" partition, even if you didn't explicitly configure it. this is also the reason I put this item second because I figured there was a hierarchy of concepts: table -> partition -> shard

but I guess from your comment, even if that is technically the case, perhaps it's more of a technicality than a useful thing to articulate in this context?

anyway, I moved it to the top and reworded it:

> CrateDB transparently segments the underlying storage of table data into shards (four by default). You can configure the number of shards with the CLUSTERED BY clause. You control how CrateDB routes table rows to shards by specifying a routing column.

side note:

this was actually the paragraph that sent me down a bit of a rabbit hole on Monday. it's really hard to describe what a shard is in simple English without a lot of text to introduce all the concepts needed to understand everything if it's entirely new to you. the two biggest problems being:

- making it clear to the reader that shards are not something that affects your typical read and write interactions with the table
- being clear when describing what a shard actually is on disk in a way that a) is technically accurate and doesn't introduce confusion with regards to the specifics of CrateDB's implementation, Elasticsearch, Lucene, etc., and b) communicates its usefulness in terms of being a "thing" at the filesystem level that can "move" (or rather, be transferred) between nodes

the sentence in my rewritten paragraph makes some assumptions re the reader's preexisting understanding that I'm not satisfied with, but it will do for now. I'd like to come back this when I do some more glossary work

Reply:
I moved the replication bullet point so it appears after this one also

makes sense to keep them sequential imo